fw/Core/Hitcon/App/SpaceshipApp.h

```diff
@@ -16,26 +16,29 @@ enum state_t { INIT, RUN, GAME_OVER, END };
 
```

Comment:
You might want to static_assert() on the display height.

Reply:
Can I use 
```
static constexpr unsigned PLANE_UPPER_BOUND = 1 << (DISPLAY_HEIGHT-2);
```
instead of using 
```
static_assert(DISPLAY_HEIGHT==8)
```